CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+## 0.52.3      2023-11-30
+
+* Update the snapshot submission log format
+  - Previously, compact (activity and log) snapshots submission log were emitted
+    every time it's completed. This was causing lots of log lines as an activity
+    snapshot submission happens every 10 seconds
+  - With this update, we group the log and only emit once per minute
```

Comment:
Another option, in an attempt to make the intent of the change clear and to generalize activity/log/compact snapshots to "real-time".

* Reduce frequency of collector log generation
  - Previously, real-time snapshots would generate log lines every 10 seconds, which made errors hard to find.
  - Now, a single log line is printed once a minute with a summary of snapshots submitted.
  - Note that `--verbose` will still log every snapshot as it's submitted.

Reply:
Nice, thanks for the suggestion. Looks much better, I'll use it 👍